app/lectures/models.py

```diff
@@ -0,0 +1,16 @@
+from django.contrib.auth import get_user_model
+from django.db import models
+from subjects.models import Subject
+
+
+# Create your models here.
+
+
+class Lecture(models.Model):
+    title = models.CharField(max_length=100, null=True, blank=True)
```

Comment:
I do not think that we want title to be null or blank. This field should always be populated